layouts/shortcodes/h3.html

```diff
@@ -0,0 +1,5 @@
+{{ $anchorized := anchorize .Inner }}
```

Comment:
I created this file initially but didn't end up using it.   I figured it would be relatively harmless to keep around in case we need it in the future (we have a very similar h2 shortcode).   I am happy to remove if there are concerns.